feature/platform/transceiver/zr_electrical_signal_to_noise_ratio_test/README.MD

```diff
@@ -0,0 +1,52 @@
+# gNMI-1.20: Telemetry: 400ZR Optics Electrical Signal to Noise Ratio(eSNR) streaming
+
+## Summary
+
+Validate 400ZR optics module reports accurate eSNR.
+eSNR is defined as the electrical Signal to Noise ratio at the decision
+sampling point in dB
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers state is enabled
+      * Both transceivers are set to a valid target TX output power
+        example -10 dBm
+      * Both transceivers are tuned to a valid centre frequency
+        example 193.1 THz
+*   With the link ZR link established as explained above, verify that the
+    following ZR transceiver telemetry paths exist and are streamed for both
+    the ZR optics
+    *   /terminal-device/logical-channels/channel/otn/state/esnr/instant
+    *   /terminal-device/logical-channels/channel/otn/state/esnr/avg
+    *   /terminal-device/logical-channels/channel/otn/state/esnr/min
+    *   /terminal-device/logical-channels/channel/otn/state/esnr/max
+
```

Comment:
Given what we just saw with one platform last week, for eSNR, CD, and other analog values we should also define what these paths should give when:
a) the device first boots up.
b) the transceiver is disabled.
c) the transceiver is enabled, but there is no signal (i.e. fiber break).

Curious on your thoughts, but for eSNR and CD, I think we could ask that (b) and (c) give a value of 0. The type here is Decimal64, so the most important part is that during these use cases, ONLY Decimal64 values are streamed. They should never stream a value such as string="nil" or string="inf". Especially, when the device is booting up, they should not stream until valid values can be streamed.

Reply:
Updated to capture these details